app/app/local.env

```diff
@@ -9,7 +9,7 @@ CACHE_URL=dbcache://my_cache_table
 REDIS_URL=rediscache://redis:6379/0?client_class=django_redis.client.DefaultClient
 CACHEOPS_REDIS=redis://redis:6379/0
 COLLECTFAST_CACHE_URL=dbcache://collectfast
-DATABASE_URL=psql://postgres:postgres@db:5432/postgres
+DATABASE_URL=psql://gitcoin_user:password@localhost:5432/gitcoin
```

Comment:
I think this change will conflict with the default docker environment setup instructions here: https://docs.gitcoin.co/mk_setup/ - Lets add this updated config as a comment to the current one with a note that it should be used if the dev is following the local setup instructions.